README.md

```diff
@@ -4,35 +4,32 @@
 
-# Contributing
+This repository tracks the ongoing status of EIPs. It contains:
 
- 1. Review [EIP-1](EIPS/eip-1.md).
- 2. Fork the repository by clicking "Fork" in the top right.
- 3. Add your EIP to your fork of the repository. There is a [template EIP here](eip-template.md).
- 4. Submit a Pull Request to Ethereum's [EIPs repository](https://github.com/ethereum/EIPs).
+- [Draft](https://eips.ethereum.org/all#draft) proposals which intend to complete the EIP review process.
+- [Last Calls](https://eips.ethereum.org/all#last-call) for proposals that may become final (see also [RSS feed](https://eips.ethereum.org/last-call.xml)).
+- [Accepted](https://eips.ethereum.org/all#accepted) proposals which are awaiting implementation or deployment by Ethereum client developers.
+- [Final](https://eips.ethereum.org/all#final) and [Active](https://eips.ethereum.org/all#active) proposals that are recorded.
+- The [EIP process](EIPS/eip-1.md#eip-work-flow) that governs the EIP repository.
 
-Your first PR should be a first draft of the final EIP. It must meet the formatting criteria enforced by the build (largely, correct metadata in the header). An editor will manually review the first PR for a new EIP and assign it a number before merging it. Make sure you include a `discussions-to` header with the URL to a discussion forum or open GitHub issue where people can discuss the EIP as a whole.
+Achieving "Final" status in this repository only represents that a proposal has been reviewed for technical accuracy. It is solely the responsibilty of the implementer to decide if these proposals will be useful to you and if they represent best practice.
 
-If your EIP requires images, the image files should be included in a subdirectory of the `assets` folder for that EIP as follows: `assets/eip-N` (where **N** is to be replaced with the EIP number). When linking to an image in the EIP, use relative links such as `../assets/eip-1/image.png`.
+Browse all current and draft EIPs on [the official EIP site](https://eips.ethereum.org/).
 
 Once your first PR is merged, we have a bot that helps out by automatically merging PRs to draft EIPs. For this to work, it has to be able to tell that you own the draft being edited. Make sure that the 'author' line of your EIP contains either your GitHub username or your email address inside <triangular brackets>. If you use your email address, that address must be the one publicly shown on [your GitHub profile](https://github.com/settings/profile).
 
-When you believe your EIP is mature and ready to progress past the draft phase, you should do one of two things:
+## Project Goal
 
- - **For a Standards Track EIP of type Core**, ask to have your issue added to [the agenda of an upcoming All Core Devs meeting](https://github.com/ethereum/pm/issues), where it can be discussed for inclusion in a future hard fork. If implementers agree to include it, the EIP editors will update the state of your EIP to 'Accepted'.
```

Comment:
This is not highlighted in index.html nor EIP-1. I think it should be added.

Reply:
Specifications for the process should be in EIP-1. This information should be out of scope for the README.